### PR TITLE
Update main.py + settings.json

### PR DESCRIPTION
**Pull request by ElBe-Plaq**
Cleaned code, created `settings.json`, added docstring, renamed many variables, deleted one use functions.

### CODE REVIEW

**Review by ElBe-Plaq — COMMENTED** [review]
Reviewed, I hope this works.

**Comment by ElBe-Plaq** [comment]
`main.exe` needs to be recompiled to use code from [8dc4ebf](https://github.com/ElBe-Plaq/rEdgeDeflect/commit/8dc4ebf7b0945e9e0a5c0a8afc2d5726c436da4f).

**Comment by ElBe-Plaq** [comment]
> `main.exe` needs to be recompiled to use code from [8dc4ebf](https://github.com/ElBe-Plaq/rEdgeDeflect/commit/8dc4ebf7b0945e9e0a5c0a8afc2d5726c436da4f).

Completed with [da4112e](https://github.com/ElBe-Plaq/rEdgeDeflect/commit/da4112e0fe7077ea5ff16b5cd58d3db5e4a15118).

**Comment by ElBe-Plaq** [comment]
I'll work on the `readme.md` file to close #6.

**Comment by Blazzycrafter** [comment]
wie war nochmal dein discord name?
nur der name der descriminator nicht...
damit wir reden können und du deinen namen nicht leakst XD

**Review by Blazzycrafter — COMMENTED** [review]
Viewed and aproved 7-9 changes...
Discuss with him before I approve the rest :D

**Comment by Blazzycrafter** [comment]
> Viewed and aproved 7-9 changes... Discuss with him before I approve the rest :D

reviewed files
ALL DELETIONS and
settings.json

**Comment by ElBe-Plaq** [comment]
> wie war nochmal dein discord name? nur der name der descriminator nicht... damit wir reden können und du deinen namen nicht leakst XD

ElBe#7741

File: main.py [hunk 1]
```diff
@@ -1,183 +1,130 @@
...
 
-def dprint(string):
-    if DEBUG:
+    Returns:
+        print: Prints the string.
+    '''
+    if DEBUG: 
         print(string)
 
-def catch(url:str):
```
Comment (Blazzycrafter) [inline main.py:63]: wo ist die den gelandet?
;D

Comment (Blazzycrafter) [reply main.py:63]: dasie einmal verwendet wurde wurde sie integriert
besprochen via dc chat

File: main.py [hunk 2]
```diff
@@ -1,183 +1,130 @@
...
-"icof" : ["--no-startup-window", "--win-session-start"],
-"Alias_Commands" : ["commands","cmd"],
-"CL" : ["microsoft-edge:"],
-"UCL" : ["http://", "https://"]
-}'''
-
-
-
-
-try:
```
Comment (Blazzycrafter) [inline main.py:18]: bin soweit zu frieden nur noch einmal das wieder rein....
sonst muss man 2 dateien herrunterladen...
ein mal die exe und dann die json...

**Comment by ElBe-Plaq** [comment]
`README.md` was created.

**Comment by ElBe-Plaq** [comment]
Is the `README.md` good?

**Comment by ElBe-Plaq** [comment]
❗ Needs to be tested.

**Comment by ElBe-Plaq** [comment]
Please do **not** merge it for now. @Blazzycrafter

**Review by Blazzycrafter — DISMISSED** [review]
Editing it a bit...
like the json :D

**Comment by Blazzycrafter** [comment]
> Please do **not** merge it for now. @Blazzycrafter

K

**Review by Blazzycrafter — COMMENTED** [review]
:D

**Comment by ElBe-Plaq** [comment]
`main.exe` needs to be recompiled.

**Comment by ElBe-Plaq** [comment]
> `main.exe` needs to be recompiled.

✔️ Done.

**Comment by Blazzycrafter** [comment]
> > `main.exe` needs to be recompiled.
> 
> ✔️ Done.

@ElBe-Plaq 
ready?

**Comment by ElBe-Plaq** [comment]
> > > `main.exe` needs to be recompiled.
> > 
> > 
> > ✔️ Done.
> 
> @ElBe-Plaq ready?

Still needs to be tested.

**Comment by Blazzycrafter** [comment]
> > > > `main.exe` needs to be recompiled.
> > > 
> > > 
> > > ✔️ Done.
> > 
> > 
> > @ElBe-Plaq ready?
> 
> Still needs to be tested.

sag wenn du soweit bist...
ambesten in discord

**Comment by ElBe-Plaq** [comment]
Can you test it? My searchbar doesn't work.

**Comment by Blazzycrafter** [comment]
K

**Comment by Blazzycrafter** [comment]
> Can you test it? My searchbar doesn't work.
__my searchbar dont like that too XD__


Loop error?
https://i.imgur.com/esITFqN.png

exit error?
https://i.imgur.com/KYQpZRf.png

asking works when i type that
https://i.imgur.com/Pm336BU.png

thats works too but we have a exit err
https://i.imgur.com/zSX0t38.png


and NO DID NOT WORK....
origenal purpose did not work...

MS is an a*s...




recherching about the urlLink microsoft edge

**Comment by ElBe-Plaq** [comment]
Try again with the new `main.exe` please.

**Comment by Blazzycrafter** [comment]
https://www.ctrl.blog/entry/microsoft-edge-protocol-competition.html

**Comment by Blazzycrafter** [comment]
ich glaube wir legen das auf eis und bauen einen Msedge killer.. XD
kills ms edge if found and uninstall its XD
and then deletes all files XD

**Comment by ElBe-Plaq** [comment]
Bruh.

**Comment by Blazzycrafter** [comment]
ich merge es und archiviere es...

**Pull request merged**